docs/src/styles/docs/home.scss

```diff
@@ -33,6 +33,7 @@
 .install-code__container {
   flex-grow: 1;
   display: flex;
+  max-width: 42rem;
 }
 
 .install-code__content {
```

Comment:
For some reason github won't let me select all the lines to make a suggestion here, but can we do something like this to keep it left aligned (and keep the $ to make it look like a shell, which isn't a huge deal without either):

```
.install-code__content {
  margin: var(--amplify-space-small);
  overflow-x: auto;
  white-space: normal;
  &:before {
    content: '$ ';
  }
}

.install-code__button {
  margin-left: auto;
  // Hiding on smaller screens
  // because there's not enough space
  @media (max-width: 480px) {
    display: none;
  }
}
```
<img width="745" alt="Screen Shot 2022-06-18 at 11 01 35 AM" src="https://user-images.githubusercontent.com/376920/174444246-d0e3e230-d5c1-406b-82cd-992bf35e57e1.png">

Reply:
I like it! Adding in that change now